R/epi_df.R

```diff
@@ -244,8 +248,10 @@ new_epi_df = function(x = tibble::tibble(), geo_type, time_type, as_of,
 #' 
 #' ex3 <- ex3_input %>% 
 #'   tsibble::as_tsibble() %>% # needed to add the additional metadata
-#'   dplyr::mutate(state = rep("MA",6)) %>%
-#'   as_epi_df(additional_metadata = c(other_keys = "state"))
+#'   dplyr::mutate(
+#'     state = rep("MA",6),
+#'     pol = rep(c("blue", "swing", "swing"), each = 2)) %>% # 2 extra keys
+#'   as_epi_df(additional_metadata = list(other_keys = "state", "pol")) 
```

Comment:
This should be `list(other_keys = c("state", "pol"))`.

Reply:
Good catch!

Reply:
Thanks for catching this! Just pushed up a fix